examples/todo/src/example.rs

```diff
@@ -0,0 +1,10 @@
+extern crate diesel;
+
+mod models;
+mod schema;
+
+use models::todo::Todo;
+
+fn main() {
+    println!("Hello, world!");
```

Comment:
In the future, we'll want to add actual DB calls using the generate code and either use an in-memory sqlite db or spin up a postgres container to make sure those calls are working

Reply:
```suggestion
    // This file (should) demonstrate the usage of the generated dsync output
    println!("Hello, world!");
    // TODO: This file in the example still needs to be expanded on
```

add a note so we dont forget the purpose